### PR TITLE
unicodecsv: add recipe

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

File: recipes/unicodecsv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+package:
```
Comment (synapticarbors) [inline recipes/unicodecsv/meta.yaml]: Repeated `package` section.

File: recipes/unicodecsv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  imports:
+    - unicodecsv
+
+about:
+  home: https://github.com/jdunck/python-unicodecsv
+  license: BSD
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (synapticarbors) [inline recipes/unicodecsv/meta.yaml]: Even though the package looks like it hasn't been updated in a while, could you submit an issue/PR mentioning the missing license file in the source distribution and put the url for the issue/PR in a comment in the recipe as a reminder to see if the situation has changed down the road.

File: recipes/unicodecsv/meta.yaml (hunk 1 above)
Comment (nehaljwani) [inline recipes/unicodecsv/meta.yaml]: Fixed.

File: recipes/unicodecsv/meta.yaml (hunk 2 above)
Comment (nehaljwani) [inline recipes/unicodecsv/meta.yaml]: Updated.

**Pull request merged**